docs/sources/enterprise/_index.md

```diff
@@ -77,44 +78,44 @@ Follow these steps to activate your Grafana Enterprise license:
 
 ### 1. Download your license file
```

Comment:
I just merged a PR that moves the license file content into another topic. Please reconcile.